docs/ref/forms/widgets.txt

```diff
@@ -157,6 +157,25 @@ this, you use the :attr:`Widget.attrs` argument when creating the widget::
         url = forms.URLField()
         comment = forms.CharField(widget=forms.TextInput(attrs={'size': '40'}))
 
+You can also modify a widget during instantiation of the form::
```

Comment:
The first example is during *definition* of the form, not *instantiation*. The second is during instantiation.